### DISCUSSION TITLE
为什么没有爱莉

### DISCUSSION THREAD

**Discussion by faetong2024** [share121/inter-knot#716] [其他]
![微信图片_20240810153540](https://github.com/user-attachments/assets/dbfd55c7-35ab-4bad-aac8-0b2f41ded6e3)

**Comment by cheeseRatRat** [comment]
![116376216_p0](https://github.com/user-attachments/assets/741ebe58-ae66-4245-b6c3-d044d5e3dcbb)

**Comment by cheeseRatRat** [comment]
![Uploading 110585210_p0.jpg…]()

**Comment by cheeseRatRat** [comment]
![Uploading 115827828_p0.png…]()

**Comment by cheeseRatRat** [comment]
![Uploading 98277883_p1.jpg…]()
